DZ2/src/timer.js

```diff
@@ -0,0 +1,68 @@
+import { formatError } from "./utils.js";
+import { HowlToExport } from './howler.js'
```

Comment:
Еще одна прелесть вебпака это то что, он нам позволяет импортировать из commonJs модулей, как будто они es-модули. Поэтому теперь можно импортировать из Howler как из Luxon)
```suggestion
import { Howl } from 'howler'
```